### PR TITLE
8297296: java/awt/Mouse/EnterExitEvents/DragWindowTest.java fails with "No MouseReleased event on label!"

### PR DESCRIPTION
**Pull request by alisenchung**
Test was run on mac, windows, linux 50 times and passed after change
Before fix, test fails about once every 50 runs on linux and windows platforms

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8297296](https://bugs.openjdk.org/browse/JDK-8297296): java/awt/Mouse/EnterExitEvents/DragWindowTest.java fails with "No MouseReleased event on label!"

### Reviewers
 * [Prasanta Sadhukhan](https://openjdk.org/census#psadhukhan) (@prsadhuk - **Reviewer**) ⚠️ Review applies to [8f0e0e71](https://git.openjdk.org/jdk/pull/11425/files/8f0e0e71132bb640405b8271c7a37871776e9488)
 * [Damon Nguyen](https://openjdk.org/census#dnguyen) (@DamonGuy - Author)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/11425/head:pull/11425` \
`$ git checkout pull/11425`

Update a local copy of the PR: \
`$ git checkout pull/11425` \
`$ git pull https://git.openjdk.org/jdk pull/11425/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 11425`

View PR using the GUI difftool: \
`$ git pr show -t 11425`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/11425.diff">https://git.openjdk.org/jdk/pull/11425.diff</a>

</details>

### CODE REVIEW

File: test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java [hunk 1]
```diff
@@ -81,6 +81,7 @@ public Point call() throws Exception {
         robot.waitForIdle();
 
         if (dragWindowMouseEnteredCount != 1) {
+//            System.out.println("dragWindowMouseEnteredCount " + dragWindowMouseEnteredCount);
```
Comment (jayathirthrao) [inline test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java:84]: We should remove or uncomment this line.

File: test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public class DragWindowTest {
     public static void main(String[] args) throws Exception {
 
         Robot robot = new Robot();
-        robot.setAutoDelay(50);
+        robot.setAutoDelay(250);
```
Comment (prsadhuk) [inline test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java:58]: We normally do not need so much delay for test to work, 100ms being the norm which has sufficed for other tests.
Maybe you need to wait robot.delay after creating and showing the GUI which is not done here.
Also, this test has heavyweight and lightweight mix, for ex. MyDragWindow class uses JPanel but not in EDT which can result in some timing issue you are seeing, either it needs to be in EDT or can it be changed to AWT Panel as done elsewhere in the test?

ALso, please change the wildcards imports while you are it..

**Review by jayathirthrao — COMMENTED** [review]
This test was de-problemlisted recently under https://bugs.openjdk.org/browse/JDK-8023562 after running test 100 times in CI. And now with increased auto delay we are trying de-problemlist it again.

With increased auto delay test is passing in our CI on all platforms when ran for 100 times, but we need to refine this test as mentioned by @prsadhuk to add needed delay at appropriate places and remove mixture of lightweight & heavyweight parts.

File: test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java [hunk 3]
```diff
@@ -65,6 +77,7 @@ public void run() {
             }
         });
 
+        robot.delay(250);
         robot.waitForIdle();
```
Comment (DamonGuy) [inline test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java:81]: Not positive if it matters, but from what I've seen the delay usually comes after waitForIdle.

But I also ran the test and it seems to be OK.

File: test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java (hunk 2 above)
Comment (prsadhuk) [reply test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java]: JPanel question is still unanswered/unchanged...

**Comment by honkar-jdk** [comment]
@alisenchung Was this an intermittent issue? If so it is better to run the entire test folder multiple times to catch any intermittent issue.

File: test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java (hunk 2 above)
Comment (alisenchung) [reply test/jdk/java/awt/Mouse/EnterExitEvents/DragWindowTest.java]: If I change the JPanel in MyDragWindow to awt Panel, the test fails

**Comment by alisenchung** [comment]
After pulling newest changes test is failing again... looking into issue

**Review by prsadhuk — APPROVED** [review]
I guess it's better to use setLocationRelativeTo() to position the frame to middle of screen too to make it more stable..

**Review by DamonGuy — APPROVED** [review]

**Comment by dholmes-ora** [comment]
@alisenchung this test continues to fail in our CI in tier4. I will file a new bug.

**Pull request closed without merging**